tests/nosetests/test_rm/test_rm.py

```diff
@@ -1,52 +1,60 @@
-"""
-"""
-
 import phoebe
 import numpy as np
 import os
-
-def _beta_vs_legacy(b, syncpar, plot=False, gen_comp=False):
-
+import matplotlib.pyplot as plt
 
 
+def _beta_vs_legacy(b, syncpar, plot=False, gen_comp=False):
     b.run_compute('phnum', model='phnumresults', overwrite=True)
     if gen_comp:
         b.run_compute('legnum', model='legnumresults', overwrite=True)
         b.filter(model='legnumresults').save('test_rm_{}.comp.model'.format(syncpar))
     else:
         b.import_model(os.path.join(os.path.dirname(__file__), 'test_rm_{}.comp.model'.format(syncpar)), model='legnumresults', overwrite=True)
 
+    phoebe1_rv1 = b.get_value('rvs@primary@legnumresults@legnum')
+    phoebe1_rv2 = b.get_value('rvs@secondary@legnumresults@legnum')
+
+    phoebe2_rv1 = b.get_value('rvs@primary@phnumresults@phnum')
+    phoebe2_rv2 = b.get_value('rvs@secondary@phnumresults@phnum')
+
+    phoebe1_rv2[np.isnan(phoebe2_rv2)] = np.nan
```

Comment:
what does phoebe1 put in these locations instead?  (mostly just curious)

Reply:
It uses the dynamical values as a drop-in replacement.